tools/net-util.py

```diff
@@ -0,0 +1,400 @@
+            'Failed P2P Connections:',
+            'Total Incoming Blocks:',
+            'Blocks Produced:',
+            'Scheduled Trxs:',
+            'Unapplied Trxs:',
+            'HTTP Requests:',
+        ]
+        self.rightFieldLabels = [
+            'Nodeos Version:',
+            'LIB:',
+            'Outbound P2P Connections:',
+            'Total Incoming Trxs:',
+            'Trxs Produced:',
+            'Blacklisted Trxs:',
+            'Dropped Trxs:',
+        ]
+        self.peerMetricConversions = {
+            'hostname': lambda x: x[1:].replace('__', ':').replace('_', '.'),
+            'port': lambda x: str(int(x)),
+            'accepting_blocks': lambda x: 'True' if x else 'False',
+            'latency': lambda x: format(int(x)/1000000, '.2f') + ' ms',
+            'last_received_block': lambda x: str(int(x)),
+            'first_available_block': lambda x: str(int(x)),
+            'last_available_block': lambda x: str(int(x)),
+            'unique_first_block_count': lambda x: str(int(x)),
+            'last_bytes_received': lambda x: str(datetime.timedelta(microseconds=(time.time_ns() - int(x))/1000)),
+            'last_bytes_sent': lambda x: str(datetime.timedelta(microseconds=(time.time_ns() - int(x))/1000)),
+        }
+        self.infoFieldLabels = [
+            'Nodeos Version ID:',
+            'Chain ID:',
+            'Nodeos Full Version:',
+            'Earliest Available Block:',
+        ]
+        self.peerColumns = [
+            ('Connection ID', 'connectionIDLW'),
+            ('\n\nIP Address', 'ipAddressLW'),
+            ('\n\nPort', 'portLW'),
+            ('\n\nHostname', 'hostnameLW'),
+            ('\n\nLatency', 'latencyLW'),
+            ('\nSend\nRate', 'sendBandwidthLW'),
+            ('Last\nSent\nTime', 'lastBytesSentLW'),
+            ('\nRcv\nRate', 'receiveBandwidthLW'),
+            ('Last\nRcv\nTime', 'lastBytesReceivedLW'),
+            ('Last\nRcvd\nBlock', 'lastReceivedBlockLW'),
+            ('Unique\nFirst\nBlks', 'uniqueFirstBlockCountLW'),
+            ('First\nAvail\nBlk', 'firstAvailableBlockLW'),
+            ('Last\nAvail\nBlk', 'lastAvailableBlockLW'),
+            ('\nAcpt\nBlks', 'acceptingBlocksLW')
+        ]
+        def labelToAttrName(label: str, fieldType='Text'):
+            return label[:1].lower() + label[1:-1].replace(' ', '') + fieldType
+        self.fields = {k:v for k, v in zip(self.leftFieldLabels, [labelToAttrName(e) for e in self.leftFieldLabels])}
+        self.fields.update({self.rightFieldLabels[0]: labelToAttrName(self.rightFieldLabels[0], 'Button')})
+        self.fields.update({k:v for k, v in zip(self.rightFieldLabels[1:], [labelToAttrName(e) for e in self.rightFieldLabels[1:]])})
+        self.fields.update({k:v for k, v in zip(self.infoFieldLabels, [labelToAttrName(e) for e in self.infoFieldLabels])})
+        
+        parser = argparse.ArgumentParser(description='Terminal UI for monitoring and managing nodeos P2P connections',
```

Comment:
it says for managing but I don't see any option other then monitoring. normally management means to alter in some specific way. So if it doesn't change nodeos in any way I suggest to remove that word it is just misleading

Reply:
The intention was to allow the tool to use the `net_api_plugin` to disconnect peers.  As that is not yet implemented, I have changed the wording.